src/evict/evict_lru.c

```diff
@@ -2748,7 +2748,11 @@ __evict_walk_tree(WT_SESSION_IMPL *session, WTI_EVICT_QUEUE *queue, u_int max_en
             /* Count internal pages queued. */
             if (F_ISSET(ref, WT_REF_FLAG_INTERNAL))
                 internal_pages_queued++;
+        } else {
+            ++page->evict_queue_attempts;
```

Comment:
I feel this counter should be incremented at the start? To find out how many times we see the same page before it is being evicted?

Reply:
done